build/utils-node.js

```diff
@@ -199,11 +215,11 @@ module.exports = {
   },
 
   getDocument: function(content) {
-    return jsdom(content);
+    return jsdom.jsdom(content, jsdom.level(3, 'core'));
```

Comment:
Could you add a comment to explain there is a problem in jsdom-nogyp so that we have to pass ```jsdom.level(3, 'core')``` as workaround